app/components/swimlanes/swimLanes.tsx

```diff
@@ -16,9 +16,9 @@ import {
 import styles from "./Swimlanes.module.css";
 import { imageURL } from "../../utils/utils";
 import { DC_URL } from "../../config/constants";
-import useBreakpoints from "../../hooks/useBreakpoints";
+import useBreakpoints from "app/hooks/useBreakpoints";
 
-const SwimLanes = ({ lanesWithNumItems }) => {
+const SwimLanes = ({ numColumns, lanesWithNumItems }) => {
   const { isLargerThanLargeTablet } = useBreakpoints();
```

Comment:
You don't need to import `useBreakpoints` here anymore.

Reply:
It's used in another place

Reply:
Ah, the tooltip.